### PR TITLE
fix(forms): emit event for nested objects

### PR DESCRIPTION
**Pull request by Leopoldthecoder**
# Summary

KM-514

#### Resources

- [Creating a package docs](https://github.com/Kong/public-ui-components/blob/main/docs/creating-a-package.md)

### CODE REVIEW

File: packages/entities/entities-plugins/src/components/PluginForm.vue [hunk 1]
```diff
@@ -864,6 +864,9 @@ const buildFormSchema = (parentKey: string, response: Record<string, any>, initi
 
     // Field type is an input, determine input type, such as 'text', or 'number'
     if (initialFormSchema[field].type === 'input') {
+      if (['string', 'number'].includes(typeof initialFormSchema[field].default)) {
+        initialFormSchema[field].placeholder = `Default: ${initialFormSchema[field].default}`
+      }
```
[the remark this answers is not in the document]
Comment (Leopoldthecoder) [reply packages/entities/entities-plugins/src/components/PluginForm.vue:869]: It seems `KMultiSelect` does not support mutating placeholder here: https://github.com/Kong/kongponents/blob/2e06005fba4b830d15974483654d75924cffd556/src/components/KMultiselect/KMultiselect.vue#L601-L607

[the remark this answers is not in the document]
Comment (Justineo) [reply packages/entities/entities-plugins/src/components/PluginForm.vue:869]: Let's add support when it's implemented.

https://kongstrong.slack.com/archives/C053JL4CE9K/p1726726743996059

**Review by sumimakito — APPROVED** [review]
Manually tested and LGTM

File: packages/entities/entities-plugins/src/components/PluginForm.vue (hunk 1 above)
[the remark this answers is not in the document]
Comment (Justineo) [reply packages/entities/entities-plugins/src/components/PluginForm.vue:869]: Added a ticket for this one: https://konghq.atlassian.net/browse/KM-526

**Pull request merged**